itest/lnd_funding_test.go

```diff
@@ -541,10 +540,28 @@ func sendAllCoinsConfirm(ht *lntest.HarnessTest, node *node.HarnessNode,
 // channel funding workflow given a channel point that was constructed outside
 // the main daemon.
 func testExternalFundingChanPoint(ht *lntest.HarnessTest) {
```

Comment:
What do you think if we leave the test as-is and create a new test `testExternalFundingSimpleTaproot`? I think it makes it easier to follow the testing flow, and easier to debug in the future since we can focus on one scenario only. This is something I've noticed when working on multi hop tests - we tried to use one test to cover all channel types, which ended up with tests that are difficult to maintain.

Reply:
Maskes sense. Wanted to avoid the overhead of starting up a whole new test case. So split things up within the same test instead. What do you think?

Reply:
Looking good!